CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ and this project now adheres to [Semantic
 Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+
+* Added support for expires date in security object.
```

Comment:
Hi @TheRealEdDawson , can you review changeLog for this one, I added a new line as codacy was expecting it , I don't think we need it but I can remove that

Reply:
Hi there @bhavya-shukla-lrn, because this file is only going to be viewed by humans, I think we can ignore the Codacy rule about line spacing in this case (it is quite silly, so I removed the blank line). I have a minor change I committed to change the choice of wording to be "expiry date". Otherwise, looks great!